tools/file_packager.py

```diff
@@ -1138,7 +1140,7 @@ def generate_js(data_target, data_files, metadata):
         if (response.ok) {
           return response.json();
         }
-        return Promise.reject(new Error(`${response.status}: ${response.url}`));
+        return handleError(new Error(`${response.status}: ${response.url}`));
```

Comment:
I think the `return Promise.reject` is needed here to avoid the `.then` firing below (i.e. to break the promise chain).

Maybe single `.catch(handleError)` on line 1146 is what is needed here to make sure we call `handleError`